integration-tests/api/__tests__/admin/discount.js

```diff
@@ -408,6 +408,7 @@ describe("/admin/discounts", () => {
             value: 10,
             allocation: "total",
           },
+          regions: ["test-region"],
```

Comment:
suggestion(non-blocking): When a string is used a lot like this, I tend to create a constant to hold it and make any changes easier and avoiding typo somewhere in the middle :) wdyt?